src/modules/previewpane/powerpreview/trace.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+class Trace
+{
+public:
+    static void RegisterProvider();
```

Comment:
1. What would happen if I call `FilePreviewerIsDisabled()` before `RegisterProvider()`?
2. Does it make sense to leverage [RAII](https://en.cppreference.com/w/cpp/language/raii) to manage the (un)registration process?